### ISSUE TITLE
Remove unneeded (and unwanted) '.' after the year in calendar repeatingCountTitle

### ISSUE THREAD

**Issue by mcavoya** [MichMich/MagicMirror#2896]
Platform: Raspberry Pi 3 (browser/Electron version agnostic)
Node Version: 16.16.0
MagicMirror² Version: 2.20.0
Description: Calendar shows a '. ' after the year in repeatingCountTitle
Steps to Reproduce: Load a calendar with repeating events. e.g. Joe's birthday, with a start date seven years ago, that repeats yearly.
Expected Results: "Joe's Birthday, 7 years"
Actual Results: "Joe's Birthday, 7. years"
Configuration: see attached [config.zip](https://github.com/MichMich/MagicMirror/files/9268902/config.zip)
Additional Notes: The fix is simple. 
Change line 265 in calendar.js
```
from: repeatingCountTitle = ", " + yearDiff + ". " + repeatingCountTitle;
to:   repeatingCountTitle = ", " + yearDiff + repeatingCountTitle;
```

In cases where puncutation is desired after yearDiff, the config file can simply add it in the repeatingCountTitle.

**Comment by rejas** [comment]
Thx for the report @mcavoya It indeed seems a little off when looking at the result in english. 
Since @MichMich wrote that line (way back in 2016) maybe he had some other i18n format in mind?

**Comment by khassel** [comment]
I think this should be fixed if there comes no veto by @MichMich

**Comment by MichMich** [comment]
All fine by me. Not emotionally attached to the `.`. ;)

**Comment by mcavoya** [comment]
I do not believe this topic should be marked Closed.

The code was refactored, but the offending '.' character remains.  It appears right after {yearDiff}.

`repeatingCountTitle = ', ${yearDiff}. ${repeatingCountTitle}';`

And it can be seen here as "31. Years"

![Screenshot 2023-04-07 070457](https://user-images.githubusercontent.com/28827793/230606610-9f1bf474-c4a5-4a6d-a6c2-f7230af00445.png)

**Comment by khassel** [comment]
you are right, sadly it was reintroduced by another PR:

![grafik](https://user-images.githubusercontent.com/25914086/230607712-7e3272e4-8725-4230-b9dc-faa9ba3b605f.png)

So I will reopen this issue and fix it again with a new PR ...

**Comment by mcavoya** [comment]
Thanks. I can only imagine that's a frequent squence of events.

**Comment by khassel** [comment]
fixed with new release `v2.24.0`
